doc/source/data_modules.rst

```diff
@@ -0,0 +1,159 @@
+.. _data_modules:
+
+Data Modules
+============
+
+Data modules are essentially directories in the :file:`data` tree. Inheritence
+rules apply similarly to the image definition tree, but additionally, `keg`
+supports cross inheritance for data modules. Cross inheritance is useful to
+inherit configuration changes from previous version. This can be specified in
```

Comment:
It either should be  "... from previous version_s_" or "...from _a_ previous version".